src/components/Screen.scss

```diff
@@ -1,9 +1,94 @@
 @import 'variables.scss';
 
-.bubble {
+.bubble-container {
     display: flex;
     justify-content: center;
     align-items: center;
     min-height: 100vh;
-    border: 1rem solid $yellow-color;
-}
\ No newline at end of file
+}
+
+.bubble-text {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 8em;
+    width: 8em;
+    border-radius: 50%;
+    font-family: roboto;
+    font-weight: 700;
+    font-size: 1.2em;
+    margin: 1em;
+}
+
+.yellow {
+    background-color: $yellow-color;
+}
+
+.violet {
+    background-color: $violet-color;
+    color: whitesmoke;
+}
+
+.green {
+    background-color: $green-color;
+}
+
+.pink {
+    background-color: $pink-color;
+}
+
+.bubble-text:hover {
+    transform: scale(2);
+    transition: 0.3s;
+}
+
+.bubble-text:hover span {
+    display: none;
+}
+
+.bubble-text.yellow:hover:before {
+    transform: scale(1.2);
+    font-size: 0.8em;
+    text-align: center;
+    transition: 0.3s;
+    content: "Creativity \A Empathy \A Responsibility \A Sociability";
+    white-space: pre-wrap;
+}
```

Comment:
this may become:

    .bubble-text:hover:before {
      transform: scale(1.2);
      font-size: 0.8em;
      text-align: center;
      transition: 0.3s;
      white-space: pre-wrap;
    }

    .bubble-text.violet:hover:before {
      content: "...";
    }

    .bubble-text.violet:hover:before {
      content: "...";
    }

be DRY (don't repeat yourself) ;)

Reply:
Changes done !